package.json

```diff
@@ -8,8 +8,27 @@
     "socket.io": "^2.0.4",
     "uuid": "^3.1.0"
   },
-
   "scripts": {
     "build": "parcel site/js/index.js site/script-nomodule.js -EM # see nathan/parcel-redux"
-  }
+  },
+  "name": "bantisocial",
+  "version": "0.0.1",
+  "description": "Actually free rip-off of Discord",
+  "main": "index.js",
+  "devDependencies": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/towerofnix/bantisocial.git"
+  },
+  "keywords": [
+    "chat",
+    "chat-system",
+    "social"
+  ],
+  "author": "Some dummies who want more than 50 custom emotes",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/towerofnix/bantisocial/issues"
+  },
+  "homepage": "https://github.com/towerofnix/bantisocial#readme"
```

Comment:
Could you add a readme of some kind in this PR? Dead links are bad!!

Reply:
Yup. Made issue #26 (so this is blocked, for now).